.kokoro/release/publish_javadoc.cfg

```diff
@@ -18,7 +18,6 @@ env_vars: {
   value: "github/java-bigquery/.kokoro/release/publish_javadoc.sh"
 }
 
```

Comment:
add 
```
# Downloads docfx doclet resource. This will be in ${KOKORO_GFILE_DIR}/<doclet name>	
gfile_resources: "/bigstore/cloud-devrel-kokoro-resources/docfx" 	
```